animations/lib/src/misc/animated_list.dart

```diff
@@ -57,7 +57,7 @@ class _AnimatedListDemoState extends State<AnimatedListDemo> {
       leading: CircleAvatar(
         child: Icon(Icons.person),
       ),
-      onLongPress: () => deleteUser(index),
+       trailing: IconButton(icon: Icon(Icons.delete), onPressed: ()=>deleteUser(index)),
```

Comment:
```suggestion
       trailing: IconButton(icon: Icon(Icons.delete), onPressed: ()=>deleteUser(index),),
```

Reply:
Please add a trailing comma here and run `flutter format .` on the project. The Travis build is failing due to a formatting problem.